core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh

```diff
@@ -50,17 +50,17 @@ add_repo_from_script() {
 	local script_url="$1"
 
 	echol "Running: curl $script_url | bash"
-	curl $script_url | bash
+	curl "$script_url" | bash
 }
 
 # params: <backup_file_path> <path_1_to_backup1> ... [path_N_to_backup]
 backup_files() {
 	local backup_file_path="$1"
 	shift
-	local paths_to_backup="$@"
+	local paths_to_backup=("$@")
 
 	# --directory='/' is for tar --verify
-	tar --create --verbose --verify --directory="/" --file="$backup_file_path" $paths_to_backup
+	tar --create --verbose --verify --directory="/" --file="$backup_file_path" "$paths_to_backup"
```

Comment:
```suggestion
	tar --create --verbose --verify --directory="/" --file="$backup_file_path" "${paths_to_backup[@]}"
```
Shouldn't be array after adding parentheses?
Expanding an array without an index only gives the first element.

Reply:
Yes, that's a good point, will change that. I missed it because it had that form before and we are using only one path to be backuped.